aws-lc-rs/src/unstable/kem.rs

```diff
@@ -51,8 +51,7 @@ use core::fmt::Debug;
 use crate::kem::Algorithm;
 use aws_lc::{NID_KYBER1024_R3, NID_KYBER512_R3, NID_KYBER768_R3};
 
-#[cfg(not(feature = "fips"))]
-pub use crate::kem::semistable::{ML_KEM_1024, ML_KEM_512, ML_KEM_768};
+pub use crate::kem::{ML_KEM_1024, ML_KEM_512, ML_KEM_768};
```

Comment:
Can these re-exports be marked deprecated without impact to the standard location? Would be nice to drop these in a subsequent release to avoid confusion on the stability aspect.